docs/iris/src/userguide/saving_iris_cubes.rst

```diff
@@ -46,12 +45,12 @@ Controlling the save process
 The :py:func:`iris.save` function passes all other keywords through to the saver function defined, or automatically set from the file extension.  This enables saver specific functionality to be called.
 
     >>> # Save a cube to PP
-    >>> iris.save(cube, "myfile.pp")
+    >>> iris.save(cubes[0], "myfile.pp")
     >>> # Save a cube list to a PP file, appending to the contents of the file
     >>> # if it already exists
     >>> iris.save(cubes, "myfile.pp", append=True)
     >>> # Save a cube to netCDF, defaults to NETCDF4 file format
-    >>> iris.save(cube, "myfile.nc")
+    >>> iris.save(cubes[0], "myfile.nc")
     >>> # Save a cube list to netCDF, using the NETCDF4_CLASSIC storage option
     >>> iris.save(cubes, "myfile.nc", netcdf_format="NETCDF3_CLASSIC")
```

Comment:
The version numbers of the netCDF format in the comment and the code are inconsistent. I don't think it matters which we go with.